gometalinter/Dockerfile

```diff
@@ -7,17 +7,17 @@ RUN go install -v github.com/alecthomas/gometalinter
 
 RUN go get -d golang.org/x/lint/golint
 WORKDIR /go/src/golang.org/x/lint/golint
-RUN git checkout 06c8688daad7faa9da5a0c2f163a3d14aac986ca # July 2nd 2018
+RUN git checkout c67002cb31c3a748b7688c27f20d8358b4193582 # Oct 11 2018
 RUN go install -v golang.org/x/lint/golint
 
 RUN go get -d github.com/opennota/check
 WORKDIR /go/src/github.com/opennota/check
-RUN git checkout d583513b5d849e66ee7c5d03c631ed2f03253680 # July 31st 2018
+#RUN git checkout 0c771f5545ff495a81aad2916c3f16add0cd3b55 # Sep 11 2018
```

Comment:
Why is this checkout and the checkout below commented?

Reply:
I forget remove the old one

Reply:
Sorry it should be uncommented, I will check it an fix in next commit